app/models/notifications/mentioned_in_comment.rb

```diff
@@ -0,0 +1,27 @@
+module Notifications
+  class MentionedInComment < Mentioned
+    def popup_translation_key
+      "notifications.mentioned_in_comment"
+    end
+
+    def deleted_translation_key
+      "notifications.mentioned_in_comment_deleted"
+    end
+
+    def self.notify(comment, recipient_user_ids)
+      # recipient_user_ids are used in Notifications::Mentioned.notify in order to filter notification
+      # recipients for private posts. It's not clear, whether this is required, since mentions filtering
+      # is performed on the post/comment creation stage and we could consider every mention object as a
+      # sufficient reason for sending a notification. The issue is that recipient_user_ids never
+      # contains parent owner id and therefore the post author doesn't receive the notification if
+      # mentioned in comment.
+      #
+      # I left intact the Notifications::Mentioned.notify and didn't investigate the message dispatcher
+      # but here I add parent_owner_id to the list so the filtering passes well.
```

Comment:
> No, that will not work either. If you use the parent-method, it will work for public posts, but for private-posts it will only send a notification, if the mentioned person is the first who receives it.

Ah, ok, that means my tests don't cover that :)

> I'm not even sure if we should extend from Mentioned, because the notify-method would be simpler (than the parent) if you completely write the logic, instead of trying to use some parent logic.

Yes, that what I probably should do.